.github/workflows/build.yaml

```
@@ -312,10 +317,126 @@ jobs:
         with:
           category: "/language:java"
 
+  archetypes:
+    runs-on: ubuntu-latest
+    needs: build
+    permissions:
+      checks: write
+      contents: read
+    strategy:
+      matrix:
+        task:
+          - {
+              "archetype": "scp-cf-spring",
+              "javaVersion": 8,
+              "startCommand": "mvn spring-boot:run -B",
+            }
+          - {
+              "archetype": "scp-cf-spring-jakarta",
+              "javaVersion": 17,
+              "startCommand": "mvn spring-boot:run -B",
+            }
+    name: ${{ matrix.task.archetype }}
```

Let's maybe add a prefix here so that it is more obvious what is actually happening in this step.

```suggestion
    name: Test Archetype ${{ matrix.task.archetype }}
```